tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -3600,6 +3600,13 @@ class HLSLExternalSource : public ExternalSemaSource {
     _In_ ExprResult &RHS,
     _In_ SourceLocation QuestionLoc);
 
+  clang::ParsedType ApplyTypeSpecSignToParsedType(
+      _In_ clang::ParsedType &parsedType,
+      _In_ TypeSpecifierSign TSS,
+      _In_ SourceLocation Loc,
+      _In_ const PrintingPolicy &policy
```

Comment:
Don't need PrintingPolicy (since you should remove Diag - but you aren't even using it here)